tests/src/unit/WebdamConfigFormTest.php

```diff
@@ -25,7 +26,6 @@ public function setUp() {
     \Drupal::setContainer($container);
   }
 
-
   public function testGetFormId() {
     $form = new WebdamConfig($this->getConfigFactoryStub());
```

Comment:
so we should create a new stub for the Webdam class I've added in the constructor?